### PR TITLE
Improve error

### PR DESCRIPTION
**Pull request by maximest-pierre**
This PR will improve the error message delivered to the user.

### CODE REVIEW

File: controller/action/AcquisitionActions.class.php [hunk 1]
```diff
@@ -115,6 +115,19 @@ protected static function email_verification($email)
     }
   }
 
+  public static function getYoutubeErrorMessage($error){
+    // Error code definition
+    $error_definition = [
+        'this email is already in use' => 'To verify this email lorem ipsum',
```
Comment (nikooo777) [inline controller/action/AcquisitionActions.class.php]: can this be changed please? Users might not understand latin :p

File: view/template/acquisition/_youtube_error.php [hunk 2]
```diff
@@ -0,0 +1,5 @@
+<?php
+
+$error_message = AcquisitionActions::getYoutubeErrorMessage($error_message ,$email);
```
Comment (kauffj) [inline view/template/acquisition/_youtube_error.php]: Stop breaking MVC! Use `prepareXXX` method. (Code search for "prepare" in an action to figure this pattern out)

File: view/template/acquisition/youtube.php [hunk 3]
```diff
@@ -43,7 +43,7 @@
 
     <div class="content">
       <?php
-      if ($error_message): echo "<div>" . "The following error occurred: ". $error_message  . " For support please send an email to hello@lbry.io" . "</div>";
+      if ($error_message): echo View::render('acquisition/_youtube_error', ['error_message' => $error_message]);
```
Comment (kauffj) [inline view/template/acquisition/youtube.php]: This is formatted oddly.

File: view/template/acquisition/youtube_status.php [hunk 4]
```diff
@@ -78,7 +78,7 @@
                         <input type="hidden" name="status_token" id="status_token" value="<?php echo $token?>"/>
                     </div>
                   <?php
-                  if ($error_message): echo "<div>" . "The following error occurred: ". $error_message  . " For support please send an email to hello@lbry.io" . "</div>";
```
Comment (kauffj) [inline view/template/acquisition/youtube_status.php]: This is formatted oddly.

File: view/template/acquisition/_youtube_error.php [hunk 5]
```diff
@@ -0,0 +1,5 @@
+<?php
+
+$error_message = AcquisitionActions::prepareYoutubeErrorMessage($error_message ,($email ?? null));
```
Comment (kauffj) [inline view/template/acquisition/_youtube_error.php]: @maximest-pierre you're still not understanding how variables get into view partials. Will discuss with you today. It should never be necessary to access a controller function directly from a view. In general the only PHP that should be appearing in templates is basic control statements (if / for) and printing variables.

**Pull request closed without merging**